packages/sdk/CONTRIBUTING.md

```diff
@@ -0,0 +1,96 @@
+# How to Contribute to the SDK
+
+## Local Environment Setup
+
+### Global Dependencies
+
+Clone the repo (ideally your own fork of it) and initialize the global
+dependencies like Node.js, NPM, etc. We use [`asdf`](https://asdf-vm.com/) to
+manage these, so assuming we're located in this directory (i.e. `packages/sdk/`)
+we can run the following command to install them:
+
+```shell
+asdf install
```

Comment:
There are also some deps at the root `package.json` (like `pnpm`) that we should probably have people install yeah?

Reply:
This command should traverse its way up the chain until the root of the repo.